tensorbay/geometry/box.py

```diff
@@ -215,6 +257,11 @@ def width(self) -> float:
         Returns:
             The width of the 2D box.
 
+        Examples:
```

Comment:
change the coordinates of the Box2D to make the width and height different values.